src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
```

Comment:
avoid raw use type

Be attentive with the diamond operator in java. What is the difference between: new Node<>(...) and new Node<K,V>(...)and new Node(...).
```suggestion
        this.table = new Node[DEFAULT_CAPACITY];
```